nsxt/policy_common.go

```diff
@@ -291,19 +298,27 @@ func getSecurityPolicyAndGatewayRulesSchema(scopeRequired bool, isIds bool, nsxI
 	if isIds {
 		ruleSchema["ids_profiles"] = getIdsProfilesSchema()
 	}
-	return &schema.Schema{
-		Type:        schema.TypeList,
-		Description: "List of rules in the section",
-		Optional:    true,
-		MaxItems:    1000,
-		Elem: &schema.Resource{
-			Schema: ruleSchema,
-		},
+	if separated {
+		ruleSchema["policy_path"] = getPolicyPathSchema(true, false, "Security Policy path")
+		ruleSchema["sequence_number"] = &schema.Schema{
+			Type:        schema.TypeInt,
+			Description: "Sequence number of the this rule",
+			Required:    true,
+		}
+		ruleSchema["context"] = getComputedContextSchema()
```

Comment:
Could you please add a comment here explaining why we are handling `context` here differently (context is required for consistency and can be derived from parent object)

Reply:
Done.